src/logger.ts

```diff
@@ -1,7 +1,8 @@
 import { configure, getLogger } from "log4js";
+import { LOG_LEVEL } from "./properties";
 
 const logger = getLogger("promise-to-file-web");
-logger.level = process.env.LOG_LEVEL || "info";
+logger.level = LOG_LEVEL;
```

Comment:
Corrected, as this was never right. Environment properties should come through the 'properties.ts' file and default values are set there too.